server/src/database/gameDB.ts

```diff
@@ -22,21 +22,31 @@ const gameComputer: Computer<BaseUnifiedGame, UnifiedGameComputed> = {
 type WatchableGame = WatchableResource<BaseUnifiedGame, UnifiedGameComputed>
 
 const gameDB: Record<string, WatchableGame> = {}
+const storage = new StoreFile<BaseUnifiedGame>('game', new RemoteStorage())
```

Comment:
Is it alright to be using `new` in a const assignment like this outside of a class?